python/tvm/relay/build_module.py

```diff
@@ -205,7 +208,7 @@ def _build_module_no_factory(mod, target=None, target_host=None, params=None, mo
     This wrapper is suitable to be used from other programming languages as
     the runtime::Module can be freely passed between language boundaries.
     """
-    return build(mod, target, target_host, params, mod_name).module
+    return build(mod, target, target_host, params=params, mod_name=mod_name).module
```

Comment:
i assume we don't need this change?

Reply:
Fixed. I acutally wanted to remove the `target_host` argument here.